Tests/kaas/plugin/run_plugin.py

```diff
@@ -44,11 +44,12 @@ def cli():
 
 @cli.command()
 @click.argument('plugin_kind', type=click.Choice(list(PLUGIN_LOOKUP), case_sensitive=False))
+@click.argument('plugin_config', type=click.Path(exists=True, dir_okay=False))
 @click.argument('clusterspec_path', type=click.Path(exists=True, dir_okay=False))
 @click.argument('clusterspec_cluster', type=str, default="default")
-def create(plugin_kind, clusterspec_path, clusterspec_cluster):
+def create(plugin_kind, plugin_config, clusterspec_path, clusterspec_cluster):
     clusterspec = load_spec(clusterspec_path)['clusters']
-    run_plugin_create(plugin_kind, clusterspec_cluster, clusterspec)
+    run_plugin_create(plugin_kind, plugin_config, clusterspec_cluster, clusterspec)
```

Comment:
I guess a similar change would be necessary for `delete`

Reply:
Applied the changes to `delete` as well